docs/build-from-scratch.md

```diff
@@ -26,6 +28,8 @@ Where `go` is your [import path][4] for Go.
 
 For Go development, it is recommended to add the Go import path (`$HOME/go` in this example) to your path.
 
+### Option 2) Release archive
```

Comment:
I think it's worth noting here that as we saw this afternoon, the standard `make` targets described in subsequent sections won't work properly if you use this option, but you can use `go build` to just build a binary.

Reply:
I should have added a bit more context in the PR description about how I ended up getting around to this implementation which wasn't quite what we discussed. Since ultimately I think we want the make targets to work as well, I wonder if rather than adding that caveat here, we shouldn't just update the Makefile to cope with building without the git repository. I'll have a look, but that will probably need some input from the team here regarding what to use in the absence of the git sha, etc. I'll have a closer look in the morning.

Reply:
Got it - that works for me. We'll just want to have that makefile PR in before our next release.

Reply:
Upon closer inspection, it is not obvious how to best handle building from an archive - and there are too many questions regarding what happens if the buildinfo fields are empty or invalid. So I will update with a note about restricting archive builds to the go build commands and update the PR.